support-frontend/assets/helpers/productCatalog.ts

```diff
@@ -465,22 +465,51 @@ export const productCatalogDescription: Record<
 	},
 };
 
-export function productCatalogDescriptionNewBenefits(
+const paperArchiveDigitalBenefit = (countryGroupId: CountryGroupId) => {
```

Comment:
all function should have action in their name.
```suggestion
const getPaperArchiveDigitalBenefit = (countryGroupId: CountryGroupId) => {
```

Reply:
Yes thats better naming, Updated.